doc/whats_new/v0.21.rst

```diff
@@ -469,7 +477,14 @@ Multiple modules
 - |Efficiency| Memory copies are avoided when casting arrays to a different
   dtype in multiple estimators. :issue:`11973` by :user:`Roman Yurchak
   <rth>`.
-
+- |Fix| refactored the `our_rand_r` function that was implemented in several
```

Comment:
Implementation is a technical detail and should not be of concern for users.
I would delete this entry, and just add :user:`Madhura Parikh <jdnc>` in the next entry.